src/test/java/com/automationpracticetest/LogInPageTest.java

```diff
@@ -0,0 +1,65 @@
+package com.automationpracticetest;
+
+import com.automationpractice.LogInPage;
+import com.automationpractice.MyAccountPage;
+import com.utils.TestListener;
+import com.utils.WebDriverSingleton;
+import com.utils.BrowserNames;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+@Listeners(TestListener.class)
+
+public class LogInPageTest extends BaseTest {
+
+    private WebDriver driver;
+    private LogInPage loginPage;
+
+    @BeforeMethod
+    public void setUp2() {
+//        driver = WebDriverSingleton.getDriver(System.getProperty("browser")); //implementation without using ENUM
+        driver = WebDriverSingleton.getDriver(BrowserNames.CHROME); //implementation for ENUM
+        loginPage = new LogInPage(driver);
+        System.out.println("BeforeMethod from LogInPageTest");
+    }
+
+    @AfterMethod
+    public void tearDown() {
+        WebDriverSingleton.killDriverInstance();
+    }
+
+    @Test
+    public void loginWithInvalidCreds() {
+        loginPage
+                .openLogInPage()
+                .logInOperation("asd@ddd.dddfd", "sdsdsdsadasdaasdeasf");
+        String logInPageFailedAuthenticationHeading1 = loginPage.getLogInPageFailedAuthenticationHeading();
```

Comment:
Thanks for this suggestion. According to our last meeting, it was decided not to use "actualSomething" <-> "expectedSomething" pattern in Asserts. Instead of this, I used the following format:
Assert.assertEquals( loginPage.getLogInPageFailedAuthenticationHeading(), "There is 1 error\nAuthentication failed.", "Error message is incorrect!!!");